stock_picking_line_sequence/views/stock_view.xml

```diff
@@ -15,13 +15,6 @@
                     <field name="sequence" widget="handle"/>
                     <field name="sequence2"/>
                 </xpath>
-                <xpath expr="//notebook/page/field[@name='move_ids_without_package']" position="attributes">
-                    <attribute name="context">{'picking_type_code': picking_type_code, 'default_picking_id': id,
```

Comment:
Why removing this ?

Reply:
It is defaulting the sequence on creation, and it seems not needed anymore.